packages/pancake-uikit/src/components/Card/CardRibbon.tsx

```diff
@@ -7,7 +7,7 @@ interface StyledCardRibbonProps extends CardRibbonProps {
 }
 
 const StyledCardRibbon = styled.div<Partial<StyledCardRibbonProps>>`
-  z-index: 1;
+  z-index: 10;
```

Comment:
The ribbon was hidden behind other elements (e.g. in finished Pool cards)
![Screenshot 2021-08-08 at 20 25 28](https://user-images.githubusercontent.com/81824236/128640285-735da913-d0ba-4381-9a7b-6fc8e39fcb70.png)